contracts/test/vault/vault.fork-test.js

```diff
@@ -152,6 +154,82 @@ forkOnlyDescribe("ForkTest: Vault", function () {
       expect(balancePreMint).to.approxEqualTolerance(balancePostRedeem, 1);
     });
 
+    it("should withdraw from and deposit to strategy", async () => {
```

Comment:
These tests could be much simpler if it just used `checkBalance` on the strategy, instead of converting to and from the underlying cTokens.

Also, since the changes are only to the vault, we shouldn't need to mint OUSD? I think the code might be clearer if we just transfer the USDC and DAI to the vault, rather than mint.

Reply:
Yup agreed, have greatly simplified this function also by adding some helper methods to more easily ready strategy balance differences. The test should read much better now.